test/views/mobileNumber/CaptureMobileNumberViewSpec.scala

```diff
@@ -58,14 +57,10 @@ class CaptureMobileNumberViewSpec extends ViewBaseSpec {
         }
 
         "have the correct field hint" in {
-          elementText(s"$fieldLabel > span.form-hint") shouldBe
+          elementText("#page-label") shouldBe
```

Comment:
These might need to be reverted as the hint text should be unchanged as far as I'm aware